app.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
+const { viewAllCategories } = require("./controller/controller");
+
+app.get("/api/categories", viewAllCategories);
+
```

Comment:
There can also be error handling added her for example if a user tries make a request to a non existant end point, which should also be tested for.

Reply:
added 404 and 500 error messages (struggling to test for 500 error though)